testApi/src/test/scala/africa/absa/testing/scapi/reporter/StdOutReporterTest.scala

```diff
@@ -81,11 +83,11 @@ class StdOutReporterTest extends FunSuite {
     val output = baos.toString
 
     // Assertions
-    assertEquals(clue(output.contains("Simple Text Report")), true)
-    assertEquals(clue(output.contains("Number of tests run: 0")), true)
-    assertEquals(clue(output.contains("Number of successful tests: 0")), true)
-    assertEquals(clue(output.contains("Number of failed tests: 0")), true)
-    assertEquals(clue(output.contains("End Report")), true)
+    assertEquals(true, clue(output.contains("Simple Text Report")))
```

Comment:
Why the change in params order?

AFAIK the signature of `assertEquals` expects the actual value to be first and the expected to be second.

```scala
  def assertEquals[A, B](
      obtained: A,
      expected: B,
      clue: => Any = "values are not the same"
  )
```

Reply:
Wrong order is a bug. Fixed in commit **TBD**.

Reply:
The signature of methods in _what_ used framework uses opposite order of parameters? MUnit?

I am looking https://scalameta.org/munit/docs/tests.html and I can see things like
`assertEquals(actual, expected) // true`

Also, looking into the code of `munit.Assertions` I see nowhere for this common order to be changed.

Could you elaborate on your reasoning that back the view of the order of obtained-expected to differ, please?

Reply:
- My experience with first "expected", then "actual" is nicely defined here - https://sourceforge.net/p/junit/mailman/message/3338997/.
- After latest refactoring no more `assertEquals` used.
- How to deal with it future?
<img width="809" alt="image" src="https://github.com/AbsaOSS/ScAPI/assets/109138042/712c952e-74da-4ab8-9cda-0a9a96eb10cd">
  - On picture you can see them munit expected 1st actual, 2nd expected.
  - We can following it when assert methods will be used.

Reply:
After a lengthy discussion offline, it seems that multiple opinions may have merit.

Reply:
After deeper analysis I have decided to follow the framework order to avoid wrong outputs.